Assets/MirageXR/Tests/NewUI/ProfileView.cs

```diff
@@ -29,23 +25,24 @@ public class ProfileView : PopupBase
     [SerializeField] private TMP_Text _txtConnectedLRS;
     [SerializeField] private TMP_Text _txtVersion;
     [SerializeField] private ClickCounter _versionClickCounter;
+    [Space]
+    [SerializeField] private PopupBase _loginViewPrefab;
```

Comment:
use a specific type instead of a base type, for easy navigation.
```suggestion
    [SerializeField] private LoginView_v2 _loginViewPrefab;
```

Reply:
fixed